dotnet/driver.go

```diff
@@ -190,12 +197,26 @@ func (c *Config) validate() error {
 	return nil
 }
 
+// validate ensures that Config and TaskConfig are cross-validated to detect any misconfigurations
+func validate(c *Config, tc *TaskConfig) (err error) {
+	if tc.RuntimeVersion != "" {
+		if slices.Contains(c.RuntimeVersions, tc.RuntimeVersion) == false {
+			return fmt.Errorf("unsupported runtime_version %s", tc.RuntimeVersion)
+		}
+
+	}
+	return
+}
+
 // TaskConfig is the driver configuration of a taskConfig within a job
 type TaskConfig struct {
 
 	// DotnetPath indicates where a dll file is found.
 	DotnetPath string `codec:"dll_path"`
 
+	// SdkVersion indicates which version of dotnet the task must be run
+	RuntimeVersion string `codec:"runtime_version"`
```

Comment:
Change it to pointer to make nullable

Reply:
Changed to pointer. thanks for mentioning it.